### ISSUE TITLE
Pixelkunst-Version des Logos

### ISSUE THREAD

**Issue by Scosh** [foss-ag/design#14]
Ich fände es cool, wenn es das Logo auch als »Pixel« Version gäbe, einfach so als netter Gimmick für nicht ganz so ernste Kontexte.

Es stellen sich direkt hierbei einige Fragen:
- für welche Auflösung/Auflösungen entscheidet man sich?
- soll man echte Rasterversionen erstellen …
- … oder Vektoren (Quadrate) an ein Grid festigen und so »Pixel« zeichnen, damit Skalierungen immer unproblematisch sind

**Comment by Scosh** [comment]
Ich finde der Vorschlag ist auf jeden Fall schon einmal ein super Anfang. Vielen Dank! 🙇 😎 🎉 

Was mich im Moment am meisten stört ist, dass der Freiraum in der Mitte nicht gleichseitig ist wie im Original:

![screenshot 2018-12-26 at 13 35 01](https://user-images.githubusercontent.com/5603235/50446073-2b7aaf80-0913-11e9-9d4d-7454ec81c8f2.png)

Das ist etwas, dass mir für die Erhaltung des Gesamtbilds wichtig wäre. Allerdings sehe ich gerade auch keine offensichtliche Lösung, wie man das hier löst, ohne das Logo anders zu verzehren … Wenn man die beiden unteren Sterne von der Mitte nach außen schiebt, muss man die Arme der Sterne künstlich verlängern … Also ich verstehe das Problem / die Schwierigkeit bei der Übersetzung in eine Pixelform.

**Comment by Scosh** [comment]
> Ich wäre dafür, eine relativ geringe Auflösung zu verwenden (das Beispiel sind jetzt 32x32 Pixel, 600% skaliert).

Ja, ich tendiere auch in diese Richtung. Immerhin soll das Logo deutlich zum Original zu unterscheiden sein. Setzt man die Auflösung _zu hoch_, ist das eher wie das Logo nochmal nachzeichnen. Allerdings stellt sich die Frage, ob man vielleicht doch noch etwas höher gehen möchte als `32x32`, damit man eben etwas mehr Spielraum hat.

> Eine Rasterversion sollte in Ordnung sein, mit den richtigen Hilfsmitteln lassen sich diese Pixelgrafiken eigentlich recht gut ohne größere Probleme skalieren, denke ich.

Das glaube ich auch, ich hab da nur Sorge, dass wenn Leute da _nicht_ drauf achten, das halt sehr scheiße aussehen kann 😅 — solche Fehler kann man man mit einer Vektorversion vielleicht besser vorbeugen, ist meine Hoffnung. Aber Fehler im Einsatz können immer passieren. Daher ist das möglicherweise der falsche Ansatz und man sollte lieber probieren eine korrekte Skalierung irgendwo zu dokumentieren.

> Wobei eine Vektorversion ja vermutlich auch nicht der größte Aufwand wäre?

Ich müsste mich da mit den Möglichkeiten von Inkscape vertraut machen. Ich kann mich gut erinnern, dass das ausnahmsweise Mal eine Sache war, die in Illustrator™️ ziemlich trivial war und sogar echt viel Spaß gemacht hat.

**Comment by fes0j** [comment]
Manchmal muss man ja etwas radikal sein, um das Mögliche zu erforschen. Ich präsentiere: die 8x8 Pixelversion:

![bit_logo8](https://user-images.githubusercontent.com/29151515/51425714-0dd1fa80-1be0-11e9-87dc-43d776eaf1aa.png)

**Comment by dragetd** [comment]
![pixelFOSS](https://user-images.githubusercontent.com/3639577/63585999-a8dce980-c5a0-11e9-9e87-41b68d251b31.png)
